the-experts-fe/src/App.js

```diff
@@ -3,16 +3,32 @@ import logo from "./logo.svg";
 import "./App.css";
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { apiResponse: "" };
+  }
+
+  callAPI() {
```

Comment:
so whats the point of this API here ? if it's just for  testing then we don't need it anymore. shouldn't you be setting up `react-router` instead ?

Reply:
it's in another issue